Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -130,6 +145,11 @@ public BoxPlotSeries()
         public bool ShowMedianAsDot { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to show the mean as a dot.
+        /// </summary>
+        public bool ShowMeanAsDot { get; set; }
```

Comment:
the property can be kept, but marked obsolete. Change the getter/setter to use the enum property.